### PR TITLE
Update swig docs

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

File: Wrapping/Java/JavaDoc.i [hunk 1]
```diff
@@ -5813,7 +5813,12 @@ passes as the initial translation to the transform. This second
 approach assumes that the moments of the anatomical objects are
 similar for both images and hence the best initial guess for
 registration is to superimpose both mass centers. Note that this
-assumption will probably not hold in multi-modality registration.   \\\\sa itk::CenteredTransformInitializer
+assumption will probably not hold in multi-modality registration.
```
Comment (zivy) [inline Wrapping/Java/JavaDoc.i]: ...assumption **is less likely** to hold in multi-modality registration.
the proposed text is slightly less conclusive as the original (for instance CT/MR registration of the head ...)

File: Wrapping/Java/JavaDoc.i [hunk 2]
```diff
@@ -6605,6 +6610,223 @@ Destructor
 public ";
 
 
+%typemap(javaimports) itk::simple::CoherenceEnhancingDiffusionImageFilter "/**
+
+Coherence enhanging diffusion and edge enhancing diffusion.
```
Comment (zivy) [inline Wrapping/Java/JavaDoc.i]: Coherence **enhancing** diffusion

File: Wrapping/Python/PythonDocstrings.i [hunk 3]
```diff
@@ -4730,7 +4730,12 @@ passes as the initial translation to the transform. This second
 approach assumes that the moments of the anatomical objects are
 similar for both images and hence the best initial guess for
 registration is to superimpose both mass centers. Note that this
-assumption will probably not hold in multi-modality registration.   \\\\sa itk::CenteredTransformInitializer
+assumption will probably not hold in multi-modality registration.
```
Comment (zivy) [inline Wrapping/Python/PythonDocstrings.i]: Same comment as in the java wrapping:
...assumption **is less likely** to hold in multi-modality registration.
the proposed text is slightly less conclusive as the original (for instance CT/MR registration of the head ...)

File: Wrapping/Python/PythonDocstrings.i [hunk 4]
```diff
@@ -5361,6 +5366,163 @@ Destructor
 ";
 
 
+%feature("docstring") itk::simple::CoherenceEnhancingDiffusionImageFilter "
+
+Coherence enhanging diffusion and edge enhancing diffusion.
```
Comment (zivy) [inline Wrapping/Python/PythonDocstrings.i]: Same comment as in java:
Coherence **enhancing** diffusion

**Review by zivy — COMMENTED** [review]
Looks good, only had two minor comments which are repeated for the java and python wrappings.

File: Wrapping/Python/PythonDocstrings.i (hunk 4 above)
Comment (blowekamp) [reply Wrapping/Python/PythonDocstrings.i]: InsightSoftwareConsortium/ITKAnisotropicDiffusionLBR#19

**Comment by blowekamp** [comment]
As the commit message indicates, these changes are automatic from the SimpleITK documentation, which may be original or taken from ITK.

File: Wrapping/Java/JavaDoc.i (hunk 1 above)
Comment (blowekamp) [reply Wrapping/Java/JavaDoc.i]: You are welcome to make a pull request to ITK for this change.

**Pull request merged**